Mathlib.lean

```diff
@@ -3067,6 +3067,7 @@ import Mathlib.RingTheory.FractionalIdeal
 import Mathlib.RingTheory.FreeCommRing
 import Mathlib.RingTheory.FreeRing
 import Mathlib.RingTheory.GradedAlgebra.Basic
+import Mathlib.RingTheory.GradedAlgebra.FGHomogeneousIdeal
 import Mathlib.RingTheory.GradedAlgebra.HomogeneousIdeal
```

Comment:
Do we need to start a new file? Can we put this PR in `GradedAlgebra.HomogeneousIdeal`?